modules/400-descheduler/DEVELOPMENT.md

```diff
@@ -0,0 +1,7 @@
+# How to generate module values
```

Comment:
Do not use this one. Let's abstain from it and just add additionalProperties: true for all deschedulers with the comment.

Reply:
Done.